_includes/styles.scss

```diff
@@ -350,7 +350,7 @@ body {
 
       a {
         text-decoration: none;
-        color: #999999;
+        color: #616161;
```

Comment:
I darkened some text to improve a PageSpeed rating, on account of the nav links being too light on a while background.